google/resource_compute_image.go

```diff
@@ -195,3 +213,9 @@ func resourceComputeImageDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceComputeImageImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	d.Set("create_timeout", computeImageCreateTimeoutDefault)
```

Comment:
Hmm, so actually I think we should stop this from being ForceNew. Just from thinking about it logically, it's the amount of time we're willing to wait for the instance to create. IMO, changing it doesn't actually mean I want to create a new one, it just means that should it recreate for some other reason, we would want to wait that long. Does that seem right? If so we can remove ForceNew and just never do a `d.Set` on it.